capa/main.py

```diff
@@ -1255,15 +1267,15 @@ def main(argv=None):
         meta = collect_metadata(argv, args.sample, args.format, args.os, args.rules, extractor)
 
         capabilities, counts = find_capabilities(rules, extractor, disable_progress=args.quiet)
-        meta["analysis"].update(counts)
-        meta["analysis"]["layout"] = compute_layout(rules, extractor, capabilities)
+
+        meta.analysis.__dict__.update(counts)
```

Comment:
> I don’t think there’s a good reason for it to be frozen.

If Frozen = True, we can't do direct item assignment, here, for example using: 

```
meta.analysis.features_counts = counts["feature_counts"]
```
for the changes I made in this PR  shows error:

```
File "pydantic/main.py", line 359, in pydantic.main.BaseModel.__setattr__
TypeError: "Analysis" is immutable and does not support item assignment
```
thus, it is updated via `__dict__.update(counts)`. If frozen is removed we won't need to update it via dict.

Reply:
lets remove frozen so that we can do the updates more easily

Reply:
Then we won't need from_capa functionality for most of the classes in result_document since we can easily update meta.

Reply:
@williballenthin  if we are removing frozen, I think we should change name FrozenModel class, as it can be misleading. Can you suggest what I should name it.

Reply:
Also can you look into script `capa_as_library.py` , since we have set meta format as MetaData everywhere. Should the output format in script as Dictionary be removed.